docs/guides/hybrid-graphics.md

```diff
@@ -72,83 +69,43 @@ This has been tested on the MacBookPro16,1 and the MacBookPro15,1. The 15,3 and
-
-    1.  Check `journalctl -k --grep=efi:`, if you don't have "efi: Apple Mac detected, using EFI v1.10 runtime services only" then you will need update your kernel (preferred) or refer this [older version](https://github.com/t2linux/wiki/blob/eb15b19c7e4d5ce79a59ff14a4bf4297a5f65edc/docs/guides/hybrid-graphics.md#enabling-the-igpu) of this page.
+    3.  Press any key other than `z` or wait, and it should boot you into Linux. If you want a silent version of this that doesn't wait for input, you can use [this fork](https://github.com/Redecorating/apple_set_os-loader).
 
-    2.  If `cat /proc/cmdline` has `efi=noruntime`, remove it from the kernel command line by editing and regenerating your bootloader config (the issue it was avoiding is fixed by newer kernels).
-
-    3.  Install the `gpu-switch` script, and then you can set NVRAM and the boot GPU from Linux.
-
-        ```sh
-        curl https://raw.githubusercontent.com/0xbb/gpu-switch/master/gpu-switch > gpu-switch
-        chmod +x gpu-switch
-        sudo chown root:root gpu-switch
-        sudo mv gpu-switch /usr/local/bin/
-        sudo gpu-switch -i
-        ```
-
-    4.  Install the apple-gmux driver from [here](https://github.com/Redecorating/apple-gmux-t2).
-
-    5.  Reboot into Linux. Display brightness should be working again if it wasn't, and `glxinfo | grep "OpenGL renderer"` should show an Intel GPU. Running programs with `DRI_PRIME=1` will make them render on your AMDGPU (some things do this automatically). You will get more battery time now as your AMD GPU can be turned off when not needed.
-
-!!! note
-    As macOS expects the dedicated GPU to be activated at startup, to avoid various display and GPU related problems (like not outputting anything to the display after sleep), switch to the dedicated GPU using `sudo gpu-switch -d` before booting into MacOS.
-    Because of this, if you want to use Linux with the iGPU again, you'll need to re run `sudo gpu-switch -i` on your Linux install and **reboot**.
+`glxinfo | grep "OpenGL renderer"` should show an Intel GPU. Running programs with `DRI_PRIME=1` will make them render on your AMD GPU (some things do this automatically). You will get more battery time now as your AMD GPU can be turned off when not needed.
 
 ## MacBookPro16,4
 
-The AMD GPU on MacBookPro16,4 is not yet compatible with Linux. As a workaround :-
+Currently the Radeon 5600M AMD GPU on MacBookPro16,4 is [not working](https://lore.kernel.org/all/3AFB9142-2BD0-46F9-AEA9-C9C5D13E68E6@live.com/) with Linux. As a workaround :-
 
 ### If you are able to edit your kernel command line :-
 
-1. This workaround recommends GRUB as the bootloader. If you want to use some other bootloader, you need to figure out how to get an equivalent of step 4 and 6 working with your bootloader.
+1. Edit the kernel command line of this boot and add the `nomodeset` kernel parameter. This will enable you to access your Linux system in safe graphics.
```

Comment:
We could use `modprobe apple-gmux force_igd=y` instead of `nomodeset`?

Reply:
That won't work if apple-set-os isn't active yet.

Reply:
Makes sense